content/GettingStarted/Deprecations.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Feature Deprecations"
+description: "Notes about CORE features that are deprecated and either receive no further updates or are scheduled for removal from TrueNAS."
+weight: 4
+---
+
+As part of security hardening and improving feature maintainability, there are occasions when TrueNAS features must be deprecated.
+Features noted in this article are either no longer receiving any maintenance or, typically due to security vulnerabilities, might be scheduled for removal from TrueNAS in a future major version.
+
+Begin planning migrations from these features immediately and note the TrueNAS upgrade paths required when a deprecated feature is in use.
+
+## S3 Service
+
+Due to security vulnerabilities and maintainability issues, the S3 service is deprecated in TrueNAS CORE 13.0 and scheduled for removal in CORE 13.1.
+Beginning in CORE 13.0-U6, the CORE web interface generates an alert when the deprecated service is in use.
```

Comment:
As discussed in person, there is a difference between `service is in use` (i.e. the service is actually running) and service is "enabled". (i.e. the service has been configured to automatically be started on boot). We handle both scenarios in our logic. Might be useful to highlight this here.

Reply:
Updated!